oxo.py

```diff
@@ -1,26 +1,48 @@
+#I don't know whats gone wrong...
 class OxoBoard:
     def __init__(self):
         """ The initialiser. Initialise any fields you need here. """
-        raise NotImplementedError("TODO: implement __init__")
+        OxoBoard.board = [[0]*3]*3
+
 
     def get_square(self, x, y):
         """ Return 0, 1 or 2 depending on the contents of the specified square. """
-        raise NotImplementedError("TODO: implement get_square")
+        return OxoBoard.board[y][x]
```

Comment:
Why `[y][x]` rather than `[x][y]`? Doesn't really matter, but the latter is a little more readable